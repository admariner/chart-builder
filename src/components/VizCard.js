// @flow
import React, { PureComponent } from 'react'
import type { Node } from 'react'
import classes from './VizCard.module.css'

export default class VizCard extends PureComponent<{ children: Node }> {
  render() {
    return <div className={classes.container}>{this.props.children}</div>
  }
}
