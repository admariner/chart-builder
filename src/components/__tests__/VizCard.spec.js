import React from 'react'
import VizCard from '../VizCard'

it('renders', () => {
  snap(
    <VizCard>
      <div>test</div>
    </VizCard>
  )
})
