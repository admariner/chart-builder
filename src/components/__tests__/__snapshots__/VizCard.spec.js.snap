// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="container"
>
  <div>
    test
  </div>
</div>
`;
